tests/src/main/java/ca/bc/gov/open/jagefilingapi/qa/backendutils/APIResources.java

```diff
@@ -3,8 +3,8 @@
 public enum APIResources {
 
     GENERATEURLAPI("/submission/generateUrl"),
-    GETCONFIGURATION("/submission/getConfiguration"),
-    EFILINGURL("https://httpbin.org");
+    USERDETAIL("/submission/"),
```

Comment:
not sure if this name is the ideal name anymore as that endpoint does not return the user detail only, it also returns the navigation details. should be `SUBMISSION` perhaps

Reply:
do we have a convention of "_"ing multi word vars. I should've asked that earlier for readability purposes.

Reply:
Modified with requested changes. Thanks @sdevalapurkar-bcgov and @akroon3r .